librz/util/path.c

```diff
@@ -12,26 +12,22 @@
 #include <rz_constructor.h>
 #include <rz_th.h>
 
-static char *portable_prefix = NULL;
-static bool portable_prefix_searched = false;
-static RzThreadLock *portable_prefix_mutex = NULL;
+RzPathPortable *rz_portable;
```

Comment:
Remove this global variable and add the struct to RzCore.

Remove constructor/destructor and it's header file and just initialize/finalize RzPath within RzCore init/fini function